fundamental-concepts/template-authorization/2_parties.md

```diff
@@ -0,0 +1,33 @@
+Like the UNIX file system has `users`, every DAML ledger has a concept of identity.
+
+Because the representation of identities are different from one ledger to another, DAML has an
+identity abstraction. This is the `Party` type. So for example if we look at the `User` template of
+the `create-daml-app` package in `daml/User.daml`{{open}}, the line
+
+```
+username: Party
+```
+
+says that the field `username` of the `User` contract will be stored on the used backing ledger in
+whatever representation that ledger uses for identities. This could be public keys or simply a
+unique identifier. It is also the backing ledger that will check the authenticity of these
+identities before any issued commands are accepted.
+
+A DAML model doesn't specify the identities for a deployment upfront, it doesn't know whether there
+will be an `Alice` or a `Bob`, or an `sshd` and a `root` on the ledger. The parties are allocated
+not in the DAML model, but by the backing ledger once the model is deployed. This is also why actual
+party identifiers don't appear in DAML contract templates.
+
+For example, here we allocate new parties `Alice`, `Bob` and `Charlie`, and create a new `User` for
+`Alice` contract on the simple in-memory ledger of the IDE:
+
+<pre class="file" data-filename="daml/User.daml" data-target="append">
+test = scenario do
+  alice <- getParty "Alice"
+  bob <- getParty "Bob"
+  charlie <- getParty "Charlie"
+  submit alice $ create User with username = alice, following = []
+</pre>
+
```

Comment:
```suggestion

If you want to see what has happened on the underlying ledger you can click on the `Scenario results` link located between lines 43 and 44 you will see the newly created `User` contract for the `Alice` `party`. 

![image](https://user-images.githubusercontent.com/56260209/84466568-ef0f5080-ac79-11ea-909b-b50c68a0bfe3.png)

```
Just a suggestion. Leave it up to you (as with all the comments) if you like it or not.